lib/spack/spack/util/parallel.py

```diff
@@ -98,9 +98,14 @@ def submit(self, fn, *args, **kwargs):
         return future
 
 
-def make_concurrent_executor() -> concurrent.futures.Executor:
-    """Can't use threading because it's unsafe, and can't use spawned processes because of globals.
-    That leaves only forking."""
-    if multiprocessing.get_start_method() == "fork":
-        return concurrent.futures.ProcessPoolExecutor(determine_number_of_jobs(parallel=True))
-    return SequentialExecutor()
+def make_concurrent_executor(
+    jobs: Optional[int] = None, *, require_fork: bool = True
+) -> concurrent.futures.Executor:
+    """Create a concurrent executor. If require_fork is True, then the executor is sequential
+    if the platform does not enable forking as the default start method. Effectively
+    require_fork=True makes the executor squential in the current process on Windows, macOS, and
```

Comment:
```suggestion
    require_fork=True makes the executor sequential in the current process on Windows, macOS, and
```

Reply:
it saves one byte and pronounced the same